Build/scripts/init_build.ps1

```diff
@@ -0,0 +1,153 @@
+#-------------------------------------------------------------------------------------------------------
+# Copyright (C) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+#-------------------------------------------------------------------------------------------------------
+
+param (
```

Comment:
Consider adding a comment here on what this script's purpose is and where it'll be called